lib/oktakit/client.rb

```diff
@@ -66,21 +66,22 @@ def get(url, options = {})
         data: options
       }
 
-      resp, status, next_page = request :get, url, **request_options
+      resp = request :get, url, **request_options
```

Comment:
Is the `@return` type in the comment above still correct?

Reply:
ah good spotting! technically it wasn't correct originally either I think